### DISCUSSION TITLE
Qt6?

### DISCUSSION THREAD

**Discussion by buhtz** [linuxdeepin/developer-center#4430] [General | 常规讨论]
Hello,
I'm upstream maintainer of [Back In Time](https://github.com/bit-team/backintime) and testing migration from PyQt5 to 6 on Debian.

One dependency in Debian is [`qt5dxcb-plugin`](https://tracker.debian.org/pkg/deepin-qt5dxcb-plugin). There is no `qt6dxcb-plugin` available.

First question is: What is that plugin and what is it good for? :smile: I'm not the founder of _Back In Time_ but just a maintainer.

Second: Is there a replacement or alternative to qt5dxcb-plugin ?

**Comment by BLumia** [comment]
If you are going to run DDE then it's required. Using Qt's `xcb` instead of our `dxcb` plugin is also okay but there might be some feature missing while using DDE or DTK-based applications. `dxcb` is also not required for plain Qt application (I assume `backintime-qt` is not based on DTK here).

We do have plans to port DDE and related projects to Qt 6 in the future. cc @kegechen
